tcms/telemetry/api.py

```diff
@@ -169,3 +169,39 @@ def _append_status_counts_to_result(count, result):
     for status in TestExecutionStatus.chart_status_names:
         status_count = count.get(status, 0)
         result.get(status).append(status_count)
+
+
+@rpc_method(name='Testing.test_case_health')
+def test_case_health(query=None):
+
+    if query is None:
+        query = {}
+
+    test_executions = TestExecution.objects.filter(**query).order_by('case_id')
+
+    data_set = []
+    entry = {'case_id': 0}
+    for test_execution in test_executions:
+        if test_execution.case_id == entry['case_id']:
+            if test_execution.status.name in TestExecutionStatus.failure_status_names:
```

Comment:
Add `BLOCKED` to `failure_status_names` (I hope this doesn't skew up anything else in the app but eitherway a blocker issue is a failure of sorts.